src/preamble.js

```diff
@@ -2153,16 +2153,28 @@ function integrateWasmJS(Module) {
 
   function getBinary() {
```

Comment:
Do we still need this getBinary() function as well, or could we just have one getBinaryPromise() function that also contains the content of this function?